dev/devicelab/lib/tasks/native_assets_test.dart

```diff
@@ -121,10 +145,11 @@ TaskFunction createNativeAssetsTest({
 }
 
 Future<int> runFlutter({
+  String command = 'run',
```

Comment:
I think we should use separate functions rather than taking an optional parameter to determine the command.

Consider renaming this function to `flutterRun` and creating a `flutterTest` that both invoke:

```dart
Future<int> _runFlutterCommand({
  required String command,
  required List<String> options,
  required void Function(String, Process) onLine);
```